src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side1;
+
+    public Square(String color, int side1) {
+        super(color);
+        this.side1 = side1;
+    }
+
+    public int getArea() {
+        return side1 * side1;
+    }
+
+    @Override
+    public String getColor() {
+        return color;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("\nFigure: Square, "
+                + "area: "
+                + getArea()
+                + " units, side: "
+                + side1
+                + " units, color:"
+                + color);
```

Comment:
Avoid using concatenation in 'System.out.println'. Instead, use 'printf' or 'String.format' for better performance and readability.